src/index.scss

```diff
@@ -0,0 +1,4 @@
+@import 'node_modules/vanilla-framework/scss/vanilla';
```

Comment:
I think this can just be `@import "vanilla"`?

Reply:
It complains that it's not able to find it and won't build when I do that. You might be mistaking it with `@include vanilla` to import the whole framework.